x-pack/filebeat/module/iptables/log/test/iptables.log-expected.json

```diff
@@ -0,0 +1,338 @@
+[
+    {
+        "@timestamp": "2019-01-08T03:37:09.000Z",
+        "destination.ip": "172.16.54.114",
+        "destination.mac": "90:10:35:5a:1e:3a",
+        "destination.port": 445,
+        "ecs.version": "1.0.0-beta2",
+        "event.dataset": "iptables.log",
+        "event.module": "iptables",
+        "fileset.name": "log",
+        "input.type": "log",
+        "iptables.ether_type": 2048,
+        "iptables.fragment_flags": "DF",
+        "iptables.id": 15743,
+        "iptables.input_device": "eth0",
+        "iptables.ip_precedence_bits": 0,
+        "iptables.ip_tos": 0,
+        "iptables.length": 52,
+        "iptables.output_device": "",
+        "iptables.tcp_flags": "SYN",
+        "iptables.tcp_reserved_bits": 0,
+        "iptables.tcp_window": 8192,
+        "iptables.ttl": 115,
+        "log.offset": 0,
+        "message": "Jan  8 03:37:09 example-host kernel: iptables DROP_INPUT: IN=eth0 OUT= MAC=90:10:35:5a:1e:3a:90:10:9e:ec:2c:71:08:00 SRC=203.0.113.36 DST=172.16.54.114 LEN=52 TOS=0x00 PREC=0x00 TTL=115 ID=15743 DF PROTO=TCP SPT=17805 DPT=445 WINDOW=8192 RES=0x00 SYN URGP=0 ",
```

Comment:
shall I keep or remove the original `message` field?

Reply:
@webmat How is this handled in the other modules?

Reply:
In some modules where there's an actual user-readable message, I grokked the structured headers and sometimes message content, and left only the user-readable message in `message`. I think that's helpful for those situations.

These firewall logs are entirely structured content, however. So I think you can remove them for now.

There's two corollaries to this:

- We've discussed in SecOps always populating `message` with an event summary, but I will leave it up to @adriansr on whether to do that or not in this PR. I think it could be tackled later, though.
- I saw some activity around a beat-wide option to keep/discard the original messages (in `event.original`, not `message`).  I think @kvch is the best person to tell you about this. For this reason, I don't think this module should handle preservation of the original explicitly.

Reply:
I decided on `log.original`, as the message could've been partially parsed by the syslog input